packages/contracts-bedrock/src/cannon/MIPS64.sol

```diff
@@ -63,8 +63,8 @@ contract MIPS64 is ISemver {
     }
 
     /// @notice The semantic version of the MIPS64 contract.
-    /// @custom:semver 1.0.0
-    string public constant version = "1.0.0";
+    /// @custom:semver 1.0.1
+    string public constant version = "1.0.1";
```

Comment:
You'll have to merge in `develop`, bump the minor (rather than patch) number again, and run `just semver-lock` from `/packages/contracts-bedrock`.